UtopiaCity/Models/CitizenAccount/Gender.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Models.CitizenAccount
+{
+    public enum Gender
+    {
+        Male=1,
```

Comment:
0 and 1 could be better, since logic will break if for some reason we get 0 from front, even if it help to detect stuff